Xamarin.Essentials/Preferences/Preferences.win32.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Runtime.Serialization.Formatters.Binary;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Preferences
+    {
+        static readonly string settingsPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), AppInfo.Name, "settings.dat");
+
+        static readonly Dictionary<string, Dictionary<string, object>> preferences = new Dictionary<string, Dictionary<string, object>>();
+
+        static Preferences()
+        {
+            if (File.Exists(settingsPath))
+            {
+                using (var stream = File.OpenRead(settingsPath))
+                {
+                    try
+                    {
+                        var formatter = new BinaryFormatter();
+                        var readPreferences = (Dictionary<string, Dictionary<string, object>>)formatter.Deserialize(stream);
+                        stream.Close();
+
+                        if (readPreferences is object)
```

Comment:
Just check for null the normal way. https://github.com/xamarin/Essentials/blob/master/CONTRIBUTING.md